src/library.js

```diff
@@ -1595,6 +1631,60 @@ LibraryManager.library = {
     stackRestore(ret);
   },
 
+#if MINIMAL_RUNTIME
+  $abortStackOverflow: function(allocSize) {
```

Comment:
how are these functions different from the "normal" ones? seems like we could use these functions for all modes?

Reply:
I tried doing that first, but it turned out to be too hard, because preamble.js JSFuncs/arrayToC+stringToC, ccall(), and postamble.js dependenciesFulfilled/runCaller() all interact with stackSave() in a way that prevented taking them out from preamble for the regular runtime. In MINIMAL_RUNTIME only JS library linkable uses exist, so it was possible to move them here.